spec/lib/mi_bridges/driver/pages_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.describe "Driver pages" do
+  MiBridges::Driver::APPLY_FLOW.each do |page|
```

Comment:
This is nice. The alternative is a shared example, maybe, but this feels better since it drives through whatever the actual flow pages are. 👍